### PR TITLE
cosmetic changes to make the code more compatible with the training component

### PR DESCRIPTION
**Pull request by verenich**
- sequence_nr -> case_id (this is a more common convention)
- *.cpickle -> *.pkl
- scripts that add extra columns (remtime, last, event_nr) are now in Python
- "Nirdizati" -> "Nirdizati Runtime" in the dashboard title

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export NODE_ENV=production
+export NODE_ENV=development
```
Comment (andreyrozumnyi) [inline deploy.sh]: why did you change it? it is mainly used for server usage.

Comment (verenich) [reply deploy.sh]: actually, this one was unintentional. I tested it on localhost and forgot to set it back

**Review by andreyrozumnyi — APPROVED** [review]
